book/src/gobject_signals.md

```diff
@@ -59,43 +67,6 @@ Whenever we now receive the "max-number-reached" signal, the accompanying number
 {{#rustdoc_include ../listings/gobject_signals/3/main.rs:signal_handling}}
 ```
 
-Because the signal handlers can be at completely different places of the code than the emitters, we have to be extra careful when dealing with RefCells and Mutexes.
```

Comment:
I think it's not ideal that this paragraph goes away. This is a common problem that people run into once they have to use `RefCell`s :) Here it's nicely explained why and what to do.

Reply:
Glad to hear that you feel that it nicely explains this topic.
I, however, was not really satisfied to begin with.
With the `Cell` changes I don't find it acceptable anymore.

It is true however, that something similar should be included again, which is why I opened https://github.com/gtk-rs/gtk4-rs/issues/392

Reply:
The thing is you're trying to over-simplify it to make it very newcomers friendly, which is not often a great idea.